lib/json_spec/cucumber.rb

```diff
@@ -18,6 +18,18 @@
   end
 end
 
+Then /^the (?:JSON|json)(?: response)?(?: at "(.*)")? with UUIDs should( not)? be:$/ do |path, negative, json|
```

Comment:
yes, I should mention about time, and "ignoring" is a better word, at first I did not want to make it too long, but the usage is not so long, so I can change that...